iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Network.java

```diff
@@ -1190,6 +1191,16 @@ default int getConnectableCount() {
      */
     BusView getBusView();
 
+    /**
+     * Get a builder to create a new VoltageAngleLimit.
+     */
+    VoltageAngleLimitAdder newVoltageAngleLimit();
+
+    /**
+     * Get all voltageAngleLimits.
+     */
+    List<VoltageAngleLimit> getVoltageAngleLimits();
```

Comment:
We could provide an implementation that filter on a specific name, but because there is no unicity constraint or a index built I don't think it is really useful to the user who can filter the list itself as efficiently.